Assignment2Conditionals/src/assignment2conditionals/Assignment2Conditionals.java

```diff
@@ -0,0 +1,107 @@
+package assignment2conditionals;
+
+import java.util.Scanner;
+
+public class Assignment2Conditionals 
+{
+
+    public static void main(String[] args) 
+    {
+     Scanner scan = new Scanner (System.in);
+           
+     System.out.println("\t*************************************\n"
+                      + "\t*       Steve's Calculator          *\n"
+                      + "\t*************************************");
+          
+     System.out.println(" 1: Math\n 2: Subtraction\n 3: Multiplication"
+                     + "\n 4: Devision\n 5: Word Counter\n 6: Exit Program\n");
+     
+  
+     System.out.print("Please Pick An Option: ");
+     int userInputNum = scan.nextInt();
+     System.out.print("\n");
+      
+     
+     if(userInputNum == 1)
+      {
+        System.out.print("Please enter in your First variable: ");
+        double var1 = scan.nextDouble();
+        System.out.print("Please enter in your Second variable: ");
+        double var2 = scan.nextDouble();
+        double product = var1 + var2;
+        System.out.println("The answer is: " + product + "\n");
+      }
+      
+     else if(userInputNum == 2)
+      {
+        System.out.print("Please enter in your First variable: ");
+        double var1 = scan.nextDouble();
+        System.out.print("Please enter in your Second variable: ");
+        double var2 = scan.nextDouble();
+        double product = var1 - var2;
+        System.out.println("The answer is: " + product + "\n");
+      }
+     
+     else if(userInputNum == 3)
+      {
+        System.out.print("Please enter in your First variable: ");
+        double var1 = scan.nextDouble();
+        System.out.print("Please enter in your Second variable: ");
+        double var2 = scan.nextDouble();
+        double product = var1 * var2;
+        System.out.println("The answer is: " + product + "\n");
+      }
+     
+     else if(userInputNum == 4)
```

Comment:
I agree with Paul about the menu items. You could also use separate System.out.println statements, one for each menu item. Either way, it makes the code easier to read and also to add/delete/edit. Cute little programs for your menu items.